http-range-header/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "http-range-header"
+version = "0.3.0"
+edition = "2018"
```

Comment:
2021 instead?

Reply:
Can't go to 2021 because we still support older rust versions :(

Reply:
ah right was thinking we could still keep tower-http on 2018 but ofc you wouldn't be able to compile it then with older versions of rust.